screeps-game-api/src/objects/impls/creep.rs

```diff
@@ -158,6 +163,14 @@ simple_accessors! {
     (ticks_to_live -> ticksToLive -> i32),
 }
 
+impl PartialEq for Creep {
+    fn eq(&self, other: &Creep) -> bool{
```

Comment:
Hope it's alright if I run `cargo fmt` on this afterwards- it'd just add spacing here between return types and the opening brace for the method.

Reply:
I tried that once. It changed so much that I reverted it. I'll endeavor to run it next time prior to making a PR...

Reply:
Ah yeah, I guess its style has probably changed quite a bit since I last ran it on this repository.

I'll run and commit the change myself. I'll do it after merging this batch of PRs, though, so that the formatting changes don't make them unmergable.